homework.md

```diff
@@ -4,101 +4,102 @@
 
 What is 2 + 2?
 
-5
+4
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12,240
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hyper Text Markup Language
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+ALtering the presentation of a web page. CSS controls the creative aspects such as colours, layout and font.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
-```css
-#h1 {
+h1 {
 
 }
-```
+
 
 ## Question 7
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language used to build websites and make them interactive.
```

Comment:
I think better way to say is to manipulate DOM Element and make website interactive.

Reply:
Thank you! I think that is a better way of saying it too :) I appreciate you taking the time to comment!